style.css

```
@@ -1,456 +1,1102 @@
+    padding-left: 24px;
+    padding-right: 24px;
+  }
+
+  section.work-section article div.main-project div.main-project-details h3 {
+    margin-top: 0;
+  }
+
+  section.work-section article div.sub-projects {
+    width: 100%;
+  }
+
+  section.work-section article div.sub-projects div.sub-project-container {
+    height: 100%;
+    padding: 50px 16px;
+    display: flex;
+    flex-direction: column;
+    justify-content: end;
+  }
+
+  section.work-section article div.first {
+    border: 2px solid black;
+  }
+
+  section.work-section article div.first div.sub-project-container h3.title {
+    display: none;
+  }
+
+  section.headline-section div.headline p {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  section.work-section article div.first div.sub-project-container p {
+    display: none;
+  }
+
+  section.work-section article div.first div.sub-project-container div {
+    display: none;
+  }
+
+  section.work-section article div.first div.sub-project-container {
+    background: url("./Images/Project-Card.png") no-repeat;
+  }
+
+  header.aboutme-heading {
+    background: url("./Images/illustration-about\ me.svg") no-repeat;
+    background-position: bottom right;
+    padding-left: 141px;
+    padding-right: 141px;
+    background-size: 44%;
+  }
+
+  section.aboutme-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  section.aboutme-section div.skills-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 30px;
+  }
+
+  section.aboutme-section div.line-aboutme {
+    width: 75%;
+  }
+
+  section.aboutme-section div.skills {
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+  }
+
+  section.headline-section ul.social-links li {
+    display: block !important;
+    margin-top: 10px;
+  }
+
+  section.aboutme-section div.skills ul li {
+    text-align: left;
+  }
+
+  section.aboutme-section div.aboutme-heading-content {
+    width: 90%;
+  }
+
+  section.headline-section {
+    flex-direction: row;
+    justify-content: start;
+    align-items: center;
+    background-position: top right;
+  }
+
+  section.headline-section ul.social-links {
+    order: 0;
+  }
+
+  section.headline-section div.headline {
+    margin-left: 5%;
+    width: 566px;
+  }
+
+  section.headline-section div.headline h1 {
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+  }
+
+  section.headline-section div.headline h2 {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+  }
+
+  nav div.nav-links {
+    display: flex;
+  }
+
+  nav div.nav-links span.icon {
+    display: flex;
+  }
```

Let's try to make our code DRY here too👍.

```suggestion
  nav div.nav-links span.icon {
    display: flex;
  }
```